src/gen_rules.ml

```diff
@@ -774,11 +774,45 @@ module Gen(P : Params) = struct
     | None -> []
     | Some name -> ["--cookie"; sprintf "library-name=%S" name]
 
+  (* Generate rules for the reason modules in [modules] and return a
+     a new module with only OCaml sources *)
+  let setup_reason_rules ~dir (m : Module.t) =
+    let refmt =
+      match Artifacts.binary "refmt" with
```

Comment:
Yes. Basically it is the same as writing `${bin:refmt}` in a user rule in a jbuild file:

- if `refmt` is a tool that is installed by the project being compiled, then jbuilder will use resolve `${bin:refmt}` to the local binary
- otherwise it is looked up in the PATH

Basically this changes nothing for users of reason. What it changes is that if you wanted to use jbuilder to build reason itself, then you could build the tests/examples directly without having to install reason first.